python/pyspark/sql/functions.py

```diff
@@ -144,12 +144,6 @@ def _():
                'measured in radians.',
 }
 
-_functions_2_2 = {
-    'to_date': 'Converts a string date into a DateType using the (optionally) specified format.',
-    'to_timestamp': 'Converts a string timestamp into a timestamp type using the ' +
-                    '(optionally) specified format.',
-}
-
```

Comment:
actually, instead of deleting this we should keep it and we should add
```
for _name, _doc in _functions_2_2.items():
    globals()[_name] = since(2.2)(_create_function(_name, _doc))
```
this is for doc tag